totalopenstation/__init__.py

```diff
@@ -1,3 +1,13 @@
 # -*- coding: utf-8 -*-
 
 __version__  = '0.5.dev'
+
+import logging
+from pprint import pformat
+
+def geo_to_debug(data):
+    logger = logging.getLogger("tops")
+    for f in data:
+        formatted = pformat(f.__geo_interface__)
+        for line in formatted.splitlines():
+            logger.debug(line.rstrip())
```

Comment:
I propose to simplify like this

```
def geo_to_debug(data):
    logger = logging.getLogger(__name__)
    for f in data:
        logger.debug(f.__geo_interface__)
```

Reply:
Function has been removed .
This could be kept for later use.